packages/cli/src/helpers/configExtractor.ts

```diff
@@ -0,0 +1,100 @@
+import Dotenv from 'dotenv';
+import _ from 'lodash';
+import { Cfgu, Config } from '@configu/sdk';
+
+type ConfigFormat =
```

Comment:
that's the old implentation

Reply:
The old implementation is what @pelegpor asked for. Once Ran is back, the `init` command will be reworked and this code will be changed accordingly.